climada/hazard/centroids/test/test_centr.py

```diff
@@ -816,6 +816,26 @@ def test_append_dif_crs(self):
         with self.assertRaises(ValueError):
             self.centr.append(centr2)
 
+    def test_append_multiple_arguments(self):
+        """Test passing append multiple arguments in the form of a list of Centroids."""
+        # create a single centroid
+        lat, lon = np.array([1, 2]), np.array([1, 2])
+        centr = Centroids(lat=lat, lon=lon)
+        # create a list of centroids
+        coords = [(np.array([3, 4]), np.array([3, 4]))]
+        centroids_list = [Centroids(lat=lat, lon=lon) for lat, lon in coords]
+
+        centr.append(*centroids_list)
+
+        self.assertEqual(centr.lat[0], 1)
+        self.assertEqual(centr.lat[1], 2)
+        self.assertEqual(centr.lat[2], 3)
+        self.assertEqual(centr.lat[3], 4)
+        self.assertEqual(centr.lon[0], 1)
+        self.assertEqual(centr.lon[1], 2)
+        self.assertEqual(centr.lon[2], 3)
+        self.assertEqual(centr.lon[3], 4)
```

Comment:
For arrays, it's more straightforward to use the numpy testing functions:
```suggestion
        np.testing.assert_array_equal(centr.lat, [1, 2, 3, 4])
        np.testing.assert_array_equal(centr.lon, [1, 2, 3, 4])
```
They will also explicitly test if the arrays have the same size etc. See https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_equal.html